test-e2e/cip-auditor/push-audit-test.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# About: Triggered by the make target "image-push-cip-auditor-e2e", this script pushes
```

Comment:
You can delete `Triggered by the make target "image-push-cip-auditor-e2e", ` phrase, because it is already discoverable if you just read the Makefile. Also, if the Makefile ever changes, this comment becomes misleading someone needs to update it again (adds to our maintenance cost).

Reply:
Ack.